tensorflow/core/kernels/reduction_gpu_kernels.cu.h

```diff
@@ -541,34 +557,34 @@ void LaunchScalarReduction(OpKernelContext* ctx, OUT_T out, IN_T in,
 template <typename T, typename Op, typename OUT_T, typename IN_T>
 void LaunchRowReduction(OpKernelContext* ctx, OUT_T out, IN_T in, int num_rows,
                         int num_cols, Op op, T init,
-                        const cudaStream_t& cu_stream) {
+                        const gpuStream_t& cu_stream) {
   if (num_cols < 1024) {
     const int threads_per_block = 128;
     const int warps_per_block = threads_per_block / 32;
```

Comment:
should it be `64` on AMD hardware?

Reply:
probably - but for now wanted to get a clean patch in that works and worry about performance later?

Reply:
i understand. but depending on the implementation of the kernel we might even have incorrect results instead of just performing slower.

Reply:
fair enough - let's see what the CI has to say. Either way I am committed to getting performance optimal on our HW next.